EnglishFilter/sections/specific.txt

```diff
@@ -262,6 +262,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 newsobserver.com##div[id^="zone-el"][isdynamictemps]
+||profitgenerator.io/pic/*$image
```

Comment:
this rule looks wide
better:
`profitgenerator.io##a[href^="/main/redirect/"] > img`